boa/src/builtins/map/mod.rs

```diff
@@ -75,7 +74,7 @@ impl Map {
         context: &mut Context,
     ) -> Result<Value> {
         if new_target.is_undefined() {
-            return context.throw_type_error("Map requires new");
+            return context.throw_type_error("Constructor Map requires 'new'");
```

Comment:
I would rephrase this a bit:

```suggestion
            return context.throw_type_error("calling the Map requires using 'new'");
```

An example of Firefox, that seems a bit more clarifying:
```
calling a builtin Map constructor without new is forbidden
```

This also requires changing the test.

Reply:
I'll be changing to Firefox's style, the one I had set was Chromium's/V8's, but I agree FF's gives more context.